config.yml

```diff
@@ -1,3 +1,6 @@
+Control:
```

Comment:
May be called it health?
Or control will have something apart from health? Like, `control/health`. Then `control/requests` to get list of requests, sent?
PUT to `/control` may change config?

Reply:
yes, it will be a control for the different purposes - changes to config included